module/files/helper.js

```diff
@@ -9,148 +9,168 @@
 // Dependencies
 const Zip = require('adm-zip');
 const fs = require('fs');
-const request = require('request');
-const awsServices = require(ROOT_PATH + '/generics/services/aws');
-const googleCloudServices = require(ROOT_PATH +
-  '/generics/services/google-cloud');
-const azureService = require(ROOT_PATH + '/generics/services/azure');
-const oracleSerices = require(ROOT_PATH + '/generics/services/oracle-cloud');
+// const request = require('request');
+// const awsServices = require(ROOT_PATH + '/generics/services/aws');
+// const googleCloudServices = require(ROOT_PATH +'/generics/services/google-cloud');
```

Comment:
comment this code as well which is present in generics/services/ all cloud provider files @VISHNUDAS-tunerlabs